assets/javascripts/discourse/initializers/global-filter-preference.js

```diff
@@ -190,14 +190,14 @@ export default {
       const filterBodyClass = `global-filter-tag-${item}`;
       if (item === tags) {
         document
-          .querySelector(`#global-filter-${item} > .btn`)
+          .querySelector(`#global-filter-${item} > a`)
```

Comment:
For followup, we usually want to use classes, so `.global-filter-button` instead of `a` here. Just for extra clarity.